tasks/change-granule-collection-pg/src/index.ts

```diff
@@ -177,8 +210,12 @@ async function changeGranuleCollectionsPG(
 
   const targetGranules = event.input.granules.filter(validateGranule);
   const oldGranulesByID: { [granuleId: string]: ValidGranuleRecord } = keyBy(oldGranules.filter(validateGranule), 'granuleId');
-  log.debug(`change-granule-collection-pg run with config ${JSON.stringify(config)}`);
-  for (const granuleChunk of chunkGranules(targetGranules, config.concurrency)) {
+
+  log.debug(`change-granule-collection-pg run with config ${JSON.stringify({
+    ...config,
+    oldGranules: undefined, // oldGranules needs to not be logged because it can be enormous
```

Comment:
I think specifying that log overflow and other hard to diagnose errors might be helpful.